src/decoders/arw.rs

```diff
@@ -75,6 +67,7 @@ impl<'a> Decoder for ArwDecoder<'a> {
       },
       _ => return Err(format!("ARW: Don't know how to decode type {}", compression).to_string()),
     };
+    let orientation = Orientation::from_tiff_ifd(&self.tiff);
```

Comment:
Isn't this a problem since the `Camera` instances are stored in a `HashMap` with no other image specific data? Doesn't two images from the same camera share the same `Camera` instance? Do we want another metadata struct instead?